website/versioned_docs/version-0.77/getting-started.md

```diff
@@ -14,8 +14,21 @@ For information around how to set up React Native, see the [React Native Getting
 
 ## Create a new React Native project
 
-Remember to call `@react-native-community/cli init` from the place you want your project directory to live.
+First, we must determine the version of `react-native-windows` you wish to use. For this example, we'll say the `latest` build since that's the latest stable release. If you run the following, you'll get the concrete version of `react-native-windows`'s `latest` tag:
 
+```bat
+npm show react-native-windows@latest version
```

Comment:
I don't think this is necessary - we only get out of sync in the delay between the when RN releases a new latest and then we release our matching latest.

Since they don't create a 0.XX-stable until **after** they release a new latest, we can't rely on that either.

I think that now that we have to use the react-native community CLI, it's easier to just call init with a hat version : `^0.77.0`